packages/web/src/scss/tools/_links.scss

```diff
@@ -1,9 +1,38 @@
+@use '@tokens' as tokens;
+@use '../settings/links';
+
+@mixin base($set-color: false) {
+    text-decoration-skip-ink: links.$text-decoration-skip-ink;
+    text-underline-offset: links.$text-underline-offset;
+
+    @if $set-color {
+        color: tokens.$action-link-primary-default;
```

Comment:
What about secondary and inverted action colors?
Shouldn't this mixin accept any of action link colors?

Reply:
There is a `dictionaries.generate-link-colors()` mixin for that.

Reply:
I'll try to improve it in next PR.